index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio_setup_and_mobile_first</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <main>
+        <header>
+            <span class="my-logo"><a href="#">My logo</a></span>
+            <nav>
+                <a href="#"><img src="./figmaImages/images/Union.png" alt="Humburger Button"></a>
+            </nav>
+        </header>
+        <section class="intro">
+            <h1 class="greeting">
+                I’m Happi.<br/>
+                Glad to see <br/> 
+                you!
+            </h1>
+            <div class="description">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through
+                     some of my work and experience! 
+                     If you like what you see and have a project you need coded, 
+                     don’t hesitate to contact me.</p>
+            </div>
+            <h2 class="connect-link">LET'S CONNECT</h2>
```

Comment:
- [ ] I think it would be better to use `a` tag here. Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using `href` ⛏🤓

Reply:
okey thanks